nautobot/core/celery/__init__.py

```diff
@@ -1,20 +1,20 @@
 import json
-import logging
 import os
 from pathlib import Path
 import pkgutil
 import sys
 
 from celery import Celery, shared_task, signals
 from celery.fixups.django import DjangoFixup
+from celery.utils.log import get_task_logger
 from django.conf import settings
 from django.core.serializers.json import DjangoJSONEncoder
 from django.utils.module_loading import import_string
 from kombu.serialization import register
 from prometheus_client import CollectorRegistry, multiprocess, start_http_server
 
 
-logger = logging.getLogger(__name__)
+logger = get_task_logger(__name__)
```

Comment:
Is this right? This is server-side code not worker-side, isn't it?

Reply:
It's correct as I see it, because the current `logger` use-cases herein are of the `import_modules` and `worker_ready` signal receivers which are each executed on the worker.